R/teal_slice.R

```diff
@@ -268,22 +268,70 @@ c.teal_slice <- function(...) {
   ans
 }
 
+#' @param x `teal_slice` object
+#' @param show_all `logical(1)` should NULL fields be returned?
+#' @keywords internal
+#' @export
+as.list.teal_slice <- function(x, show_all = TRUE) {
+  checkmate::assert_class(x, "teal_slice")
+
+  x <- if (shiny::isRunning()) {
+    shiny::reactiveValuesToList(x)
+  } else {
+    shiny::isolate(shiny::reactiveValuesToList(x))
+  }
+
+  formals <-
+    if ('expr' %in% names(x)) {
```

Comment:
Use `class`. `filter_expr` produces a different one to that returned by `filter_var`.

Reply:
64fd3b181a0a7ad6211fcf1a195dced703804960